### ISSUE TITLE
gophish is crashing on html template

### ISSUE THREAD

**Issue by drewbeebe** [gophish/gophish#423]
I see in crash message below that there's a bad character.
Which HTML template is it referring to? HTML template for the landing page? 
Or the HTML template for the email? (I'm using HTML in my email, as well).

2016/11/09 15:20:07 worker.go:90: template: html_template:12: unexpected bad character U+003C '<' in command
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
        panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x61a4a5]

**Comment by jordan-wright** [comment]
Are you building from source or using a pre-built binary? When is the error occurring - is it when sending the emails, or is it when someone hits the landing page?

**Comment by jordan-wright** [comment]
Hey @drewbeebe, just wanted to check in on this. Are you building gophish from source, or using a pre-built binary?

**Comment by drewbeebe** [comment]
Sorry. It is a prebuilt binary and, while I fixed it my simplifying the HTML, it should be noted that complex HTML pasted in might be giving gophish some problems in the email field.

-Drew

**Comment by jordan-wright** [comment]
Yeah, it looks like this is the result of "smart pasting", as opposed to pasting as plaintext. I'll see if there are CKEditor options to fix this for HTML tag characters (<>/).
